tests/openGoogle.js

```diff
@@ -24,7 +24,7 @@ const testGoogle = async () => {
   await input.press('Enter');
   // await page.waitForNavigation({ waitUntil: 'domcontentloaded' });
   await recorder.stopScreenshots();
-  await recorder.createVideos();
+  await recorder.createStepVideo(3000, 1);
```

Comment:
Please, add a comment, explaining why `3000` is passed here as a parameter. The code doesn't give an obvious answer to that